middleware/kubernetes/kubernetes.go

```diff
@@ -221,12 +233,9 @@ func (k *Kubernetes) Records(name string, exact bool) ([]msg.Service, error) {
 		return nil, errNoItems
 	}
 
-	// TODO: Implementation above globbed together segments for the serviceName if
-	//       multiple segments remained. Determine how to do similar globbing using
-	//		 the template-based implementation.
-	namespace = k.NameTemplate.NamespaceFromSegmentArray(serviceSegments)
-	serviceName = k.NameTemplate.ServiceFromSegmentArray(serviceSegments)
-	typeName = k.NameTemplate.TypeFromSegmentArray(serviceSegments)
+	serviceName = serviceSegments[0]
+	namespace = serviceSegments[1]
+	typeName = serviceSegments[2]
```

Comment:
I don't see the code here, but I get uncomfortable when I see accesses to an array without bounds checking...